pkg/lib/cockpit-components-table.jsx

```diff
@@ -169,7 +171,7 @@ export const ListingTable = ({
         setActiveSortDirection(direction);
     };
 
-    const rowsComponents = (isSortable ? sortRows() : rows).map((row, rowIndex) => {
+    const rowsComponents = (isSortable ? (sortMethod ? sortMethod(rows, activeSortDirection, activeSortIndex) : sortRows()) : rows).map((row, rowIndex) => {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17830-20221130-151606-eb9be7bf-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)